syntax = "proto3";

message ModelConfig {
    string model_dir = 1;
    string checkpoint_path = 2;
    string meta_graph_path = 3;
    int32 intra_op_parallelism_threads = 4;
    int32 inter_op_parallelism_threads = 5;
    bool enable_mkl = 6;
    int32 kmp_blocktime = 7;
    bool kmp_settings = 8;
    string kmp_affinity = 9;
    bool enable_xla = 10;
    bool enable_tensorrt = 11;
    string trt_plan_model_path = 12;
    string trt_uff_model_path = 13;
    bool enable_fp16 = 14;
    bool enable_tpu = 15;
    string tpu_name = 16;
    string tpu_model_path = 17;
}
